start_bundle_coupled.R

```diff
@@ -478,24 +461,35 @@ for(scen in common){
         start_now <- TRUE
     }
     foldername <- file.path("output", fullrunname)
-    if ((i > start_iter_first | !start_magpie) && file.exists(foldername)) {
+    if ((i > start_iter_first || !scenarios_coupled[scen, "start_magpie"]) && file.exists(foldername)) {
       if (! "--test" %in% flags) unlink(foldername, recursive = TRUE, force = TRUE)
       message("Delete ", foldername, if ("--test" %in% flags) " if not in test mode", ". ", appendLF = FALSE)
       deletedFolders <- deletedFolders + 1
     }
+
+    if (cfg_rem$gms$optimization == "nash" && cfg_rem$gms$cm_nash_mode == "parallel") {
```

Comment:
maybe: rename `number_of_regions` to `numberOfNodes` or so? Because it is not the number of regions, but rather the number of CPUs to be used (and, potentially, for your test environment, you might want to set that to 1 as well).